script/generate-mlperf-inference-submission/customize.py

```diff
@@ -209,6 +209,7 @@ def generate_submission(i):
             implementation = sut_info["implementation"]
             device = sut_info["device"]
             framework = sut_info["framework"].replace(" ","_")
+            framework_version = sut_info["framework_version"]
```

Comment:
For framework_version and run_config we can take "default" as default if not given by the user.